### PR TITLE
Release for  0.2.0

### PR DESCRIPTION
**Pull request by lalitb**
Release 0.2.0
## Changes

New alpha release 0.2.0

As of now, it's ok to have same semver for both api and sdk, and hence single source package release. Once we start deviating in release number, we can plan to have separate source packages for api, and sdk  ( including exporters, propagators, instrumentations )

For significant contributions please make sure you have completed the following items:

* [X] `CHANGELOG.md` updated for non-trivial changes
* [ ] Unit tests have been added
* [X] Changes in public API reviewed

### CODE REVIEW

**Review by ThomsonTan — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,7 +15,14 @@ Increment the:
 
 ## [Unreleased]
 
-* Added `ForceFlush` to `TracerProvider` in SDK. ([#588](https://github.com/open-telemetry/opentelemetry-cpp/pull/588)).
+## [0.2.0] 2021-03-02
+
+* [SDK] Added `ForceFlush` to `TracerProvider` in SDK. ([#588](https://github.com/open-telemetry/opentelemetry-cpp/pull/588)).
+* [SDK] Added Resource API
```
Comment (reyang) [inline CHANGELOG.md:21]: Consider putting the actual PR link (similar like what we have for #588).

**Review by reyang — APPROVED** [review]
LGTM.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,7 +15,14 @@ Increment the:
 
 ## [Unreleased]
 
-* Added `ForceFlush` to `TracerProvider` in SDK. ([#588](https://github.com/open-telemetry/opentelemetry-cpp/pull/588)).
+## [0.2.0] 2021-03-02
+
+* [SDK] Added `ForceFlush` to `TracerProvider` in SDK. ([#588](https://github.com/open-telemetry/opentelemetry-cpp/pull/588)).
```
Comment (reyang) [inline CHANGELOG.md:20]:
```suggestion
* [SDK] Added `ForceFlush` to `TracerProvider`. ([#588](https://github.com/open-telemetry/opentelemetry-cpp/pull/588)).
```

File: CHANGELOG.md (hunk 1 above)
Comment (lalitb) [reply CHANGELOG.md]: Good point. Done now.

**Pull request merged**